sdcm/nemesis.py

```diff
@@ -4800,6 +4804,32 @@ def clean_unbootstrapped_node():
             self.log.info("Decommission added new node")
             self.cluster.decommission(new_node)
 
+    def disrupt_disable_binary_and_gossip(self):
```

Comment:
Please change the name to reflect it includes also a major compaction.

Reply:
Done